apps/meteor/client/views/marketplace/AppsList/AppRow.tsx

```diff
@@ -7,12 +7,12 @@ import type { KeyboardEvent, MouseEvent, ReactElement } from 'react';
 import React, { memo } from 'react';
 import semver from 'semver';
 
-import AppAvatar from '../../../../components/avatar/AppAvatar';
+import AppAvatar from '../../../components/avatar/AppAvatar';
 import AppStatus from '../AppDetailsPage/tabs/AppStatus/AppStatus';
 import AppMenu from '../AppMenu';
 import BundleChips from '../BundleChips';
 
-type AppRowProps = App & { isMarketplace: boolean };
+type AppRowProps = App;
```

Comment:
????

Reply:
The isMarketplace flag became obsolete sometime during the development of the new marketplace and I forgot to remove this Props type. Fixed.